src/providers/command-composer-view.provider.ts

```diff
@@ -121,19 +122,17 @@ export class CommandComposerViewProvider implements vscode.WebviewViewProvider {
 
           additionalControl = `<select class='${selectClass} ${additionalControlClasses}' id="${key}" key="${key}" child="${parentId}" title="${key}" aria-label="${option}">`;
 
-          if (optionType === "string") {
```

Comment:
Why this was reduced? 🤔
What if option type is `PwScripts` and what about properties `key`, `value` and `script` and `selected`?

Reply:
Also in some cases it produces empty script select:
![obraz](https://github.com/user-attachments/assets/ce2fc0cf-1a00-47c8-b595-9c58e3fc8c7c)
while its expected to have eg:

![obraz](https://github.com/user-attachments/assets/13ff9bdd-b719-421c-9681-bc193ec771d8)

(from v1.2.13)

Reply:
You are right, the code has been restored. :)